actiontext/lib/action_text/fixture_set.rb

```diff
@@ -62,7 +62,7 @@ def self.attachment(fixture_set_name, label, column_type: :integer)
       signed_global_id = ActiveRecord::FixtureSet.signed_global_id fixture_set_name, label,
         column_type: column_type, for: ActionText::Attachable::LOCATOR_NAME
 
-      %(<action-text-attachment sgid="#{signed_global_id}"></action-text-attachment>)
+      %(<#{Attachment.tag_name} sgid="#{signed_global_id}"></#{Attachment.tag_name}>)
```

Comment:
I think this should be fully qualified: `ActionText::Attachment` just to avoid confusion with other constants (maybe not that ruby would get it wrong, but other devs?).

Also what about adding a test?
https://github.com/rails/rails/blob/main/actiontext/test/unit/fixture_set_test.rb

Reply:
@zzak I'd attempted to add test coverage, but at the point the `test` block executes, the fixtures are already loaded and so any test-level overrides (like a`ActionText::Attachment.with(tag_name: "at-attachment") { ... }`) occur too late in the process.

Is there a way for tests to re-load fixtures?

Reply:
Thanks for taking a look.

> Is there a way for tests to re-load fixtures?

Probably we could make this work with #50428 but would need to investigate